packages/design-system-shared/src/utils/caip-address.test.ts

```diff
@@ -0,0 +1,280 @@
+import {
```

Comment:
Adding tests to ensure address are returned the same regardless of their format EVM, CAIP etc